.github/scripts/pr_comment.js

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
Is there no similarly-functional thing here that is at least in Python? It's annoying enough to have to maintain our own build scripts in-repo rather than importing them from elsewhere, but to have to do it in a foreign programming language (with, ironically, no test coverage of its own) feels like it's likely to become a huge maintenance headache.

Reply:
This can be implemented in Python. No problem.

I went with GitHub API helper since it's easy to experiment.

I have some local JS testing harness that helped me write this... but I am not proud of that code...so I have not added it here.

I didn't want to write a lot of code and tests for something that we decided that we don't want.

Testing will most probably involve a lot of injections since this makes heavy use of GitHub API.

If we decide that using a single comment is how we want to receive the coverage report, I can look into rewriting this in Python.